bigquery/google/cloud/bigquery/client.py

```diff
@@ -1127,6 +1126,82 @@ def job_from_resource(self, resource):
             return job.QueryJob.from_api_repr(resource, self)
         return job.UnknownJob.from_api_repr(resource, self)
 
+    def create_job(
+        self, job_config, source=None, destination=None, query=None, retry=DEFAULT_RETRY
+    ):
+        """Create a new job.
+        Arguments:
+            job_config (dict): configuration job representation returned from the API.
+
+        Keyword Arguments:
+            source (Union[ \
```

Comment:
These arguments are quite odd to be including in a generic "create job" function.

Better to add the needed properties to the relevant -JobConfig classes.